recipes/plantcv/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "plantcv" %}
+{% set version = "3.0.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "e8524b622379cf3df03fa1a9a87b7f754b1942329581cc7d94e9d93a3793ca4f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - setuptools
```

Comment:
I guess all of this can be removed from the host section and only python and setuptools needs to be there.